app/api/items/get_prerequisites.go

```diff
@@ -0,0 +1,173 @@
+package items
+
+import (
+	"net/http"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/database"
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// swagger:model prerequisiteItem
+type prerequisiteItem struct {
+	*commonItemFields
+
+	// item_dependencies.score
+	// required: true
+	Score int `json:"score"`
+	// required: true
+	GrantContentView bool `json:"grant_content_view"`
+
```

Comment:
Probably it would be useful to add a comment on these attributes to make clear they are related with dependencies and not with items or results.

Reply:
Re-reading it, probably calling them `dependency_required_score` and `dependency_grant_content_view` would make thing clearer.

Reply:
fixed